storage/tianmu/core/tianmu_table.cpp

```diff
@@ -610,10 +610,9 @@ void TianmuTable::Field2VC(Field *f, loader::ValueCache &vc, size_t col) {
     case MYSQL_TYPE_SHORT:
     case MYSQL_TYPE_LONG:
     case MYSQL_TYPE_INT24:
-    case MYSQL_TYPE_LONGLONG: {
+    case MYSQL_TYPE_LONGLONG:
+    case MYSQL_TYPE_BIT: {
       int64_t value = f->val_int();
-      common::PushWarningIfOutOfRange(m_tx->Thd(), std::string(f->field_name), value, f->type(),
```

Comment:
Why rm this limit function? It will cause sth. wrong when we use unsigned data here.